similarity_search.py

```diff
@@ -0,0 +1,37 @@
+import os
+import numpy as np
+from sklearn.metrics.pairwise import cosine_similarity
+
+# load file path environment variable
+embeddings_file_path = os.environ(PATH_TO_EMBEDDINGS)
```

Comment:
PATH_TO_EMBEDDINGS is undefined.

Please structure the part of the code to reduce duplication and unnecessary code changes.

Either by moving the logic into a single module or converting it into a lib.
For such a small project. It's better to go with a single file module approach.